### DISCUSSION TITLE
AUTOMATIC SNAP! (BIG AFK AGATHA DECK)

### DISCUSSION THREAD

**Discussion by HolidayPhenom** [AdriaGual/marvel-snap-bot#75] [General]
My "afk agatha" deck has been doing fair. Wanted to know how I would go about making AUTOMATIC SNAP a feature on the bot for mindless decks with high winrates.

BIG AFK AGATHA

# (2) Domino
# (3) Wave
# (4) Jubilee
# (5) Devil Dinosaur
# (6) Doctor Doom
# (6) Odin
# (6) America Chavez
# (6) Hulk
# (6) Agatha Harkness
# (6) Giganto
# (6) The Infinaut
# (9) Death

# Copy the full code below and paste it from the deck editing menu in Snap:

eyJDYXJkcyI6W3siQ2FyZERlZklkIjoiRG9taW5vIn0seyJDYXJkRGVmSWQiOiJEckRvb20ifSx7IkNhcmREZWZJZCI6IkFnYXRoYUhhcmtuZXNzIn0seyJDYXJkRGVmSWQiOiJXYXZlIn0seyJDYXJkRGVmSWQiOiJKdWJpbGVlIn0seyJDYXJkRGVmSWQiOiJEZXZpbERpbm9zYXVyIn0seyJDYXJkRGVmSWQiOiJPZGluIn0seyJDYXJkRGVmSWQiOiJBbWVyaWNhQ2hhdmV6In0seyJDYXJkRGVmSWQiOiJIdWxrIn0seyJDYXJkRGVmSWQiOiJHaWdhbnRvIn0seyJDYXJkRGVmSWQiOiJJbmZpbmF1dCJ9LHsiQ2FyZERlZklkIjoiRGVhdGgifV19

If anyone knows how to a code for the bot to play in locations that are not winning already for turns 5 and 6 (assuming WAVE gets played on turn 3) let me know. Again the main feature I'm trying to get is AUTOMATIC SNAP though.

**Comment by ZloyPupsic** [comment]
this is bot inception lol

**Reply by paperc07** [reply]
Define great, cause I asked about this and owner said it wasn't working good. What rank are you at? Also why American  Chavez when it's gonna play Agatha turn 6 no matter what

**Reply by HolidayPhenom** [reply]
Switched it from ZOO to BIG. I let it play overnight and barely was catching GOLD. Highest I've been is Diamond though so I'm still working my way back up.

**Comment by HolidayPhenom** [comment]
A code to keep Jubilee from being played on the left location would also be great (at turn 5 mainly, if Agatha was played at 4 on curve after Wave) considering she keeps locking out Giganto lol

**Comment by AdriaGual** [comment]
This deck got me to gold this MODOK season, all the games played by the bot, it's a zoo dracula (it usually keeps the infinaut and red skull).

![image](https://user-images.githubusercontent.com/25803231/218343814-8595ff4b-3e0f-4bd9-be91-f9d6ab865ae7.png)
![image](https://user-images.githubusercontent.com/25803231/218343878-b8b26858-9ad8-410b-8279-ce3a63bd89c5.png)

Nothing great or fancy but hey, it does the job haha
